xline/src/storage/index.rs

```diff
@@ -49,6 +61,34 @@ impl Index {
         rev.filter(|kr| !kr.is_deleted())
             .map(KeyRevision::as_revision)
     }
+
+    /// Insert `KeyRevision` of deleted and generate `Revision` pair of deleted
+    fn gen_del_revision(
+        revs: &mut Vec<KeyRevision>,
+        revision: i64,
+        sub_revision: i64,
+    ) -> Option<(Revision, Revision)> {
+        let last_rev = Self::get_revision(revs, 0)?;
+        let del_rev = KeyRevision::new_deletion(revision, sub_revision);
+        revs.push(del_rev);
+        Some((last_rev, del_rev.as_revision()))
+    }
+
+    /// Mark the `KeyRevision` as available
+    pub(crate) fn mark_available(&self, revision: i64) {
```

Comment:
The `mark_available` is only being used in the `kv_store`, so there is no point to describe it as crate visibility, super is enough.

Reply:
Fixed